HyperVExtension/test/HyperVExtension/HyperVExtension.UnitTest.csproj

```diff
@@ -9,6 +9,7 @@
     <Nullable>enable</Nullable>
     <UseWinUI>true</UseWinUI>
     <ProjectPriFileName>resources.pri</ProjectPriFileName>
+    <EnableUnsafeBinaryFormatterSerialization>true</EnableUnsafeBinaryFormatterSerialization>
```

Comment:
is this needed?

Reply:
Unfortunately, yes. And I will need to add it to HyperV extension when DevSetupAgent deployment will be called from it. PowerShell direct remoting uses it and doesn't work without it. PS (Hyper-V?) folks should fix it. (If they already did, I haven't found it).